src/test/java/com/puppycrawl/tools/checkstyle/checks/header/RegexpHeaderCheckTest.java

```diff
@@ -204,123 +204,123 @@ public void testFailureForMultilineRegexp() throws Exception {
     }
 
     @Test
     public void testRegexpHeaderSmallHeader() throws Exception {
         final DefaultConfiguration checkConfig =
                 createCheckConfig(RegexpHeaderCheck.class);
-        checkConfig.addAttribute("headerFile", getPath("regexp.header2"));
+        checkConfig.addAttribute("headerFile", getPath("InputRegexpHeader2.header"));
         checkConfig.addAttribute("multiLines", "3, 6");
         final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
-        verify(checkConfig, getPath("InputRegexpSmallHeader.java"), expected);
+        verify(checkConfig, getPath("InputRegexpHeaderSmallHeader.java"), expected);
     }
 
     @Test
     public void testEmptyMultiline()
             throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(RegexpHeaderCheck.class);
-        checkConfig.addAttribute("headerFile", getPath("regexp.header2"));
+        checkConfig.addAttribute("headerFile", getPath("InputRegexpHeader2.header"));
         checkConfig.addAttribute("multiLines", "");
         final String[] expected = {
             "1: " + getCheckMessage(MSG_HEADER_MISSING),
         };
-        verify(checkConfig, getPath("InputRegexpSmallHeader.java"), expected);
+        verify(checkConfig, getPath("InputRegexpHeaderSmallHeader.java"), expected);
     }
 
     @Test
     public void testRegexpHeaderMulti52()
             throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(RegexpHeaderCheck.class);
-        checkConfig.addAttribute("headerFile", getPath("regexp.header3"));
+        checkConfig.addAttribute("headerFile", getPath("InputRegexpHeader3.header"));
         final String[] expected = {
             "1: " + getCheckMessage(MSG_HEADER_MISSING),
         };
-        verify(checkConfig, getPath("InputRegexpHeader5.java"), expected);
+        verify(checkConfig, getPath("InputRegexpHeaderMulti52.java"), expected);
     }
 
     @Test
     public void testIgnoreLinesSorted() throws Exception {
         final DefaultConfiguration checkConfig =
                 createCheckConfig(RegexpHeaderCheck.class);
-        checkConfig.addAttribute("headerFile", getPath("regexp.header5"));
+        checkConfig.addAttribute("headerFile", getPath("InputRegexpHeader5.header"));
         checkConfig.addAttribute("multiLines", "7,5,3");
         final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
-        verify(checkConfig, getPath("InputRegexpHeader8.java"), expected);
+        verify(checkConfig, getPath("InputRegexpHeaderIgnoreLinesSorted.java"), expected);
     }
 
     @Test
     public void testHeaderWithInvalidRegexp() throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(RegexpHeaderCheck.class);
-        checkConfig.addAttribute("headerFile", getPath("regexp.invalid.header"));
+        checkConfig.addAttribute("headerFile", getPath("InputRegexp.invalid.header"));
```

Comment:
Missed the full file name.
`InputRegexp.invalid.header` => `InputRegexpHeader.invalid.header`